ArduCopter/takeoff.cpp

```diff
@@ -106,6 +106,8 @@ void Mode::auto_takeoff_run()
     if (!motors->armed() || !copter.ap.auto_armed) {
         // do not spool down tradheli when on the ground with motor interlock enabled
         make_safe_ground_handling(copter.is_tradheli() && motors->get_interlock());
+        pos_control->relax_velocity_controller_xy();
+        pos_control->update_xy_controller();
```

Comment:
maybe there should be a relax_z_controller() call here as well?  Maybe some or all of these?

        pos_control->relax_velocity_controller_xy();
        pos_control->update_xy_controller();
        pos_control->relax_z_controller(0.0f);   // forces throttle output to decay to zero
        pos_control->update_z_controller();
        attitude_control->reset_yaw_target_and_rate();
        attitude_control->reset_rate_controller_I_terms();
        attitude_control->input_thrust_vector_rate_heading(pos_control->get_thrust_vector(), auto_yaw.rate_cds());